src/app/components/pages/QuestionFinder.tsx

```diff
@@ -115,29 +114,31 @@ export const QuestionFinder = withRouter(({location}: RouteComponentProps) => {
     const [excludeBooks, setExcludeBooks] = useState<boolean>(!!params.excludeBooks);
     const [searchDisabled, setSearchDisabled] = useState(true);
 
-    const [populatedUserContext, setPopulatedUserContext] = useState(false);
-
-    useEffect(function populateFromUserContext() {
-        if (isAda && isLoggedIn(user) && user.registeredContexts && user.registeredContexts.length > 1) {
-            setSearchStages([STAGE.ALL]);
-            setSearchExamBoards([EXAM_BOARD.ALL]);
-        }
-        else  {
-            if (!STAGE_NULL_OPTIONS.includes(userContext.stage)) {
-                setSearchStages(arr => arr.length > 0 ? arr : [userContext.stage]);
-            }
-            if (!EXAM_BOARD_NULL_OPTIONS.includes(userContext.examBoard)) {
-                setSearchExamBoards(arr => arr.length > 0 ? arr : [userContext.examBoard]);
+    const [populatedFromAccountSettings, setPopulatedFromAccountSettings] = useState(false);
+    useEffect(function populateFiltersFromAccountSettings() {
+        if (isLoggedIn(user)) {
+            const filtersHaveNotBeenSpecifiedByQueryParams = !params.stages && !params.examBoards;
```

Comment:
I think this is the only blocking change. Anything else like removing the meaningless "all" tag can be made into its own card, since I think we're deciding that having no filters applied when logged out is fine. 

```suggestion
            const filtersHaveNotBeenSpecifiedByQueryParams = Object.keys(params).every(param => !["stage","examBoard"].includes(param) ? !params[param] : true);
```

There are probably other ways to handle this line, but this seems clean to me.